### PR TITLE
ABPI-83848

### PR DESCRIPTION
**Pull request by mmiesbach**
- Details are displayed when the user clicks on a device from the list
- Fragment shows the name and mac address of the device
- The user is able to navigate back to the list from that view

### CODE REVIEW

**Comment by mmiesbach** [comment]
I didn't use a ViewModel for the DetailsFragment. Not sure if it is necessary in here...

File: .idea/misc.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
       <map>
```
Comment (flowolf86) [inline .idea/misc.xml:5]: the .idea folder you could add to your .gitignore file

File: app/src/main/java/com/example/bluetoothdevicesapplication/ui/view/DevicesRecyclerViewAdapter.kt [hunk 2]
```diff
@@ -47,8 +49,8 @@ class DevicesRecyclerViewAdapter(private val pairedDevices: List<BluetoothDevice
         }
 
         override fun onClick(p0: View?) {
-            TODO("Not yet implemented - Navigate to Details Page")
-
+            val action = DevicesFragmentDirections.actionDevicesFragmentToDetailFragment(pairedDevices[adapterPosition].name, pairedDevices[adapterPosition].address)
```
Comment (flowolf86) [inline app/src/main/java/com/example/bluetoothdevicesapplication/ui/view/DevicesRecyclerViewAdapter.kt:52]: well done

File: app/src/main/java/com/example/bluetoothdevicesapplication/ui/view/MainActivity.kt [hunk 3]
```diff
@@ -2,11 +2,28 @@ package com.example.bluetoothdevicesapplication.ui.view
...
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var navController: NavController
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        val navHostFragment = supportFragmentManager.findFragmentById(R.id.nav_host_fragment) as NavHostFragment
+        navController = navHostFragment.findNavController()
```
Comment (flowolf86) [inline app/src/main/java/com/example/bluetoothdevicesapplication/ui/view/MainActivity.kt]: You could try to make this a Kotlin extension if you like to get into that (optional)

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,22 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (flowolf86) [inline app/src/main/res/layout/activity_main.xml]: I'd continue using the ConstraintLayout, it's an interesting learning experience. Seems complex at first, but when you get how it works, it's really cool

**Review by flowolf86 — APPROVED** [review]
Nice!

**Comment by flowolf86** [comment]
Soon I need to give you more complex tasks 👍

**Pull request merged**